### DISCUSSION TITLE
Colour Scheme

### DISCUSSION THREAD

**Discussion by tristanlbailey10** [tristanlbailey10/Tafe230220#4] [General]
Lindsey, do you have a preference as to the colour scheme for our feature pages in the project?

To clarify, the colour scheme would address the colours used on controls in a page, as well as the background colour for the page itself.

**Comment by LindseyBianco** [comment]
Yeah, I'm not too fussed.
I generally prefer a dark mode or theme, how does that sound.

**Reply by tristanlbailey10** [reply]
Sounds fine by me! :)
